### DISCUSSION TITLE
Load only clicked tabs after restart Firefox?

### DISCUSSION THREAD

**Discussion by Nagamichisan** [sienori/Tab-Session-Manager#827] [Q&A]
Hi.
After opening Firefox, ALL tabs load automatically. Is it possible to set that only clicked tabs load?
Regards, Markus

**Comment by Nagamichisan** [comment]
Hi, no idea anyone?

**Comment by Nagamichisan** [comment]
:( There seems to be no solution?

**Comment by susancolvine** [comment]
The closest solution would be to enable "delayed tab loading" in TSM's settings, and then click on the tabs you want to load first. Otherwise, I'm not sure. 

Maybe there exists a different extension for this kind of thing?
